### PR TITLE
feat(Raven Code): compatibility with new website design

### PR DESCRIPTION
**Pull request by skyhancloud**
## Description 
Compatibility with new website design.

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary>Proof showing the creation/modification is working as expected</summary>

image
image

</details>

### CODE REVIEW

**Comment by EncryptedDev** [comment]
Resolve conflicts

**Comment by chimpdev** [comment]
> Resolve conflicts

how?

**Comment by EncryptedDev** [comment]
> > Resolve conflicts
> 
> how?

By pressing 'Resolve conflicts' below or through VSC source control

**Comment by chimpdev** [comment]
image
💀

File: websites/R/Raven Code/dist/metadata.json [hunk 1]
```diff
@@ -1,23 +1,19 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.7",
 	"author": {
-		"name": "can",
-		"id": "613700645173592086"
+		"name": "skyhan",
+		"id": "949705281863032872"
```
Comment (EncryptedDev) [inline websites/R/Raven Code/dist/metadata.json:5]: Have you been in contact with the old presence author to authorise this change? If you haven't got permission, please add yourself as a contributor instead

File: websites/R/Raven Code/presence.ts [hunk 2]
```diff
@@ -22,31 +22,49 @@ presence.on("UpdateData", async () => {
...
 			break;
 
 		case "/codes":
 			presenceData.details = "Kodlar sayfasını görüntülüyor";
-			if (searchParams.get("filter") !== "all") {
-				presenceData.state = `Arama: ${decodeURIComponent(
-					searchParams.get("filter")
-				)} — Sayfa ${searchParams.get("page")}`;
-			} else presenceData.state = `Sayfa ${searchParams.get("page")}`;
+			if (searchParams.get("search") !== null)
```
Comment (EncryptedDev) [inline websites/R/Raven Code/presence.ts:65]:
```suggestion
			if (searchParams.get("search"))
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/R/Raven Code/dist/metadata.json (hunk 1 above)
Comment (chimpdev) [reply websites/R/Raven Code/dist/metadata.json:5]: That's my account. It was disabled for some reason I don't understand.

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**